rom-hacking/scripting/animation-commands/ac-down.mdx

```diff
@@ -0,0 +1,40 @@
+# (769) _AC_DOWN
+
+## Effect
+
+Moves the actor of the animation down over a given amount of frames per tile.
+
+If the second argument is negative, 16 is used, or 8 if the player's DashFlag is set. (See EvEntityCommand$$GetWalkTime)
```

Comment:
You can remove the "See EvEntityCommand..." mention here, this was just a personal note. It might be worth mentioning that more research is needed on this "DashFlag" in the info blurb lower down. (I have no idea what currently sets it)